src/data/__factories__/blocks.js

```diff
@@ -0,0 +1,1061 @@
+/* eslint-disable import/prefer-default-export */
+
+// Course Blocks API response for the demo course.
+export const blocksAPIResponse = {
```

Comment:
Yes, I can make a small course and export the blocks from that, but this is most reproducible data set since it's based on the demo course. 

This data isn't likely to change, and in has enough content in it to support other tests in the future.